Application Actions/TIDAL/1.0/Play or Pause.scpt

```diff
@@ -0,0 +1,5 @@
+tell application "System Events"
+	tell process "TIDAL"
+		keystroke " "
```

Comment:
Sure thing.

On Wed, Aug 31, 2016, 13:49 Lars Alexander Blumberg <
notifications@github.com> wrote:

> In Application Actions/TIDAL/1.0/Play or Pause.scpt
> <https://github.com/getsenic/nuimo-app-osx-actions/pull/20#discussion_r76973000>
> :
>
> > @@ -0,0 +1,5 @@
> > +tell application "System Events"
> > +	tell process "TIDAL"
> > +		keystroke " "
>
> For the sake of consistency, can you replace with key code 49?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/getsenic/nuimo-app-osx-actions/pull/20/files/59030b525e0b69b031b2671032f36def5451e62f#r76973000>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAFvylg1LW7MWpL2OkvHlXnrtKNvlIiks5qlWpDgaJpZM4Jwym3>
> .
>